documentation/asciidoc/computers/raspberry-pi/eeprom-bootloader.adoc

```diff
@@ -187,6 +187,18 @@ The number of times that SD boot will be retried after failure before moving to
 
 Default: `0`
 
+[[SD_OVERCURRENT_CHECK]]
+==== `SD_OVERCURRENT_CHECK`
+
+Before booting, the bootloader checks the SD power switch overcurrent signal. The overcurrent signal occurs if the SD card is damaged and has a short circuit which will cause it to get hot.
```

Comment:
Is this for all Pis, or only for certain models of Pi?

Reply:
All Pis with a bootloader EEPROM so no need to specify a subset here

Reply:
Actually scrap that. I forgot that CMs and Pi400 can't do this.